docs/Gallery.md

```diff
@@ -35,3 +35,10 @@ This page highlights some visualizations of SpECTRE simulations.
 Here's an image of an adaptively refined mesh:
 
 <img src="https://drive.google.com/thumbnail?id=1ivR2Wbu_RYvHih098waKi6jqx637pSUn&sz=w600"/>
+
+## Binary Black Holes
+
+Visualization of apparent horizons of an equal mass non-spinning BBH. The
+colorful surface represents the lapse and the arrows depict the shift vector.
+
+<img src="https://drive.google.com/thumbnail?id=1Ccz_ZDPTpIsLXh_lGotGBdSLEnc7a__f=w1200"/>
```

Comment:
The `&sz` is missing. It's part of the query syntax of the URL. Should be `<FILE_ID>&sz=w1200`